pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -32,7 +32,7 @@ import (
 
 func (c *Compiler) createProcess(container *yaml_types.Container, stepType backend_types.StepType) (*backend_types.Step, error) {
 	var (
-		uuid = ulid.Make()
+		uuid string
 
```

Comment:
The uuid is **always** required!! We should not make it optional.

Overwrite it with an external/existing uuid is ok through

Reply:
Yes but if we add that option. It add the posibility to create a bug wich result in strange behaviour in completly unrelated parts. I rather have it created and stored even if not jet jused, as to have a posibility to create a bug in the future, because some refactorung did not had this in mind.

Anyway i think i should also add a integrity check to the backend data ... i open a new issue